### DISCUSSION TITLE
Create a idaes flowsheet from dataframe or json file?

### DISCUSSION THREAD

**Discussion by arunksagotra** [IDAES/idaes-pse#1371] [Ideas]
Hi Team,

 I would like to ask that is there a method to generate an IDAES flowsheet from either a DataFrame or a JSON file?

**Comment by andrewlee94** [comment]
@arunksagotra We do have some model serialization tools that can read and write json files, however these are probably not what you are looking for (these are serializations of the full model data including variable values, etc). You could write one of these yourself, but I am not sure that it would be an easy exercise.

Other than that however, we do not have any tools for generating a flowsheet from data. I am not sure how easy (or useful) this would be either due to the large amount of information that needs to be stored and the fact that IDAES models are built from Python classes.
